src/js/tabs/firmware_flasher.js

```diff
@@ -1,16 +1,13 @@
 import { i18n } from '../localization';
 
 const firmware_flasher = {
-    releases: null,
-    releaseChecker: new ReleaseChecker('firmware', 'https://api.github.com/repos/betaflight/betaflight/releases'),
-    jenkinsLoader: new JenkinsLoader('https://ci.betaflight.tech'),
-    gitHubApi: new GitHubApi(),
+    targets: null,
+    releaseLoader: new ReleaseLoader('https://build.betaflight.com'),
```

Comment:
Wondering what is the risk here of dropping old way to get firmware? Should we keep both at least for first release/experimental flag?

Reply:
The new system supports the old hexs.